lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
Didn't we discuss also changing Ruby regexps? Any reason you needed to do this for Python and not for Ruby (or other languages)? Should all of our regexps have anchors?